File: sales-api/Responses/SalesResponse.cs
using Sales.Models;
using Sales.Structs;

namespace Sales.Responses
{
    public class SalesResponse : ReturnStruct
    {
        public SalesResponse()
        {
            listCategories = new();
            listProducts = new();
            listBrands = new();
            listSales = new();
        }

        public List<Category> listCategories { get; set; }
        public List<Product> listProducts { get; set; }
        public List<Brand> listBrands { get; set; }
        public List<Sale> listSales { get; set; }
    }
}

File: sales-ui/src/app/sales.response.ts
import { BrandModel } from "./brand.model";
import { CategoryModel } from "./category.model";
import { ProductModel } from "./product.model";
import { ReturnStruct } from "./return-struct.response";
import { SaleModel } from "./sale.model";

export class SalesResponse extends ReturnStruct {
    public listCategories: CategoryModel[];
    public listProducts: ProductModel[];
    public listBrands: BrandModel[];
    public listSales: SaleModel[];
}